Gemfile

```diff
@@ -2,6 +2,10 @@
 # Pin CocoaPods Version to avoid that bugs in CocoaPods like
 # https://github.com/CocoaPods/CocoaPods/issues/12081 break our release
 # workflow.
-# 1.16.2 has a bug that breaks the release workflow
+# 1.16.2 has a bug that breaks the release workflow. Publishing with craft then
+# fails with:
+# - ERROR | [iOS] unknown: Encountered an unknown error (Unable to locate the
+#   executable `rsync`) during validation.
```

Comment:
reading the error message -- do you not just want to install rsync?

Reply:
yes that would be the place -- you can build the docker image locally and play around with it too

Reply:
I will do that before updating CocoaPods. Thanks.